sites/cocacola.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+const scraper = require("../peviitor_scraper.js");
+const uuid = require("uuid");
+
+const url = "https://careers.coca-colahellenic.com/ro_RO/careers/SearchJobs/?1293=%5B6003%5D&1293_format=2880&listFilterMode=1&projectRecordsPerPage=50";
+
+const company = { company: "Coca Cola HBC" };
```

Comment:
company name must not contain spaces

```suggestion
const company = { company: "CocaColaHBC" };
```

Reply:
Done